apps/dexdrip/dexdrip.c

```diff
@@ -227,7 +227,7 @@ uint32 getSrcValue(char srcVal) {
 }
 void print_packet(Dexcom_packet* pPkt) {
     uartEnable();
-    printf("%lu %hhu %d", dex_num_decoder(pPkt->raw), pPkt->battery, adcConvertToMillivolts(adcRead(0)));
+    printf("%lu %hhu %d\n", dex_num_decoder(pPkt->raw), pPkt->battery, adcConvertToMillivolts(adcRead(0)));
```

Comment:
That's fine! People hoping to use this code in embedded applications (like
myself) need termination chars for serial communications.

On Tuesday, June 2, 2015, Stephen Black <notifications@github.com> wrote:

> In apps/dexdrip/dexdrip.c
> <https://github.com/StephenBlackWasAlreadyTaken/wixel-xDrip/pull/33#discussion_r31564119>
> :
>
> > @@ -227,7 +227,7 @@ uint32 getSrcValue(char srcVal) {
> >  }
> >  void print_packet(Dexcom_packet* pPkt) {
> >      uartEnable();
> > -    printf("%lu %hhu %d", dex_num_decoder(pPkt->raw), pPkt->battery, adcConvertToMillivolts(adcRead(0)));
> > +    printf("%lu %hhu %d\n", dex_num_decoder(pPkt->raw), pPkt->battery, adcConvertToMillivolts(adcRead(0)));
>
> unfortunately this would likely cause issues with what the xdrip app
> itself is expecting, if you have a space before the newline it may be
> alright, but Im not positive
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/wixel-xDrip/pull/33/files#r31564119>
> .
>